compiler/pipes/code-gen.cpp

```diff
@@ -176,6 +194,25 @@ void CodeGenF::prepare_generate_function(FunctionPtr func) {
   }
 }
 
+std::string CodeGenF::shorten_occurence_of_class_in_file_name(ClassPtr occuring_class, const std::string &file_name) {
```

Comment:
Don't you want to add a label at the beginning of the cpp file, with its original name? Or save it to some file.

This is information to be useful for `kphp-tools`.

Reply:
(discussed offline)